CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ add_subdirectory(3rdparty/catch2)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Woverloaded-virtual")
 set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_DEBUG_POSTFIX "d")
```

Comment:
This looks unnecessary. At least it does not feel related to the search bar ;)

Reply:
resolve in commit 9344a35